core/controllers/subtopic_viewer_test.py

```diff
@@ -82,7 +82,8 @@ def setUp(self):
                     'en': {
                         'filename': 'test.mp3',
                         'file_size_bytes': 100,
-                        'needs_update': False
+                        'needs_update': False,
+                        'duration': 0.34234
```

Comment:
Can we add a unit to the duration field name, here and everywhere else in this PR? E.g. duration_msecs or duration_secs or duration_mins or similar (I can't tell what the 0.34234 here represents).

Reply:
Yeah I can change it to reflect secs, so duration_secs.

Reply:
Done.